public/forms/validation/form.ttl

```diff
@@ -79,9 +76,12 @@ ext:formValidationL a form:Listing;
   sh:group ext:formFieldPg;
   sh:order 3 .
 
-ext:formValidationValueItem a form:SubForm;
-  form:includes fields:validatonSelectorF;
-  form:hasFieldGroup ext:formFieldValidations .
+ext:formValidationValueItem a form:SubForm ;
+  form:hasFieldGroup ext:validationsFG ;
+  form:includes ext:validationResultMessageF .
```

Comment:
It seems that we _need at least 1 form:includes_ in a subform for it to render.. this is most likely a bug in the addon. It works out here since I think we _always_ want the error message field?